easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb

```
@@ -0,0 +1,35 @@
+##
+# Authors:: 
+# * Thekla Loizou <t.loizou@cyi.ac.cy>
+# * Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'ConfigureMake'
+
+name = 'MATIO'
+version = '1.5.21'
+
+homepage = 'https://sourceforge.net/projects/matio/'
+description = """matio is an C library for reading and writing Matlab MAT files."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+github_account = 'tbeu'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['2b09738fc765609794cdd4745649dcb2d7a528e9ef427c3b8e2a0d71d37d02f7']
+
+preconfigopts = './autogen.sh && '
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Autotools', '20200128'),
```

I think this should be:
```suggestion
    ('Autotools', '20200321'),
```